token/hmac/hmacsha_test.go

```diff
@@ -131,3 +132,17 @@ func TestValidateWithRotatedKeyInvalid(t *testing.T) {
 
 	require.EqualError(t, new(HMACStrategy).Validate(token), "a secret for signing HMAC-SHA512/256 is expected to be defined, but none were")
 }
+
+func TestCustomHMAC(t *testing.T) {
```

Comment:
Could you please also add a success case? :)

Reply:
Yes, sure.